packages/simulator/lib/dispatch/dispatchCentral.js

```diff
@@ -63,6 +69,15 @@ const dispatch = (cars, bookings) => {
         filter((b) => b.length > 0),
         //mergeMap((bookings) => getVroomPlan(cars, bookings)),
         mergeMap(async (bookings) => {
+          if (bookings.length < cars.length) {
+            return [
```

Comment:
This fixes a bug where we should not try to optimise a situation where there are more cars than packages.